packages/@coorpacademy-app-review/src/views/slides/index.ts

```diff
@@ -108,6 +116,31 @@ const isLastSlideAnswered = (slidesRef: string[], slideRef: string): boolean =>
   return last(slidesRef) === slideRef;
 };
 
+const getSlideMedia = (
+  state: StoreState,
+  slideFromAPI: SlideFromAPI
+): MediaPropsForPlayer | VideoPropsForPlayer | void => {
+  const media = get(['question', 'medias', '0'], slideFromAPI) as SlideMedia | void;
+  if (!media) return;
+  const {type} = media;
+  const resource = get(['src', '0'], media);
+  switch (type) {
+    case 'img':
+    case 'audio': {
+      const mediaProps: MediaPropsForPlayer = {
+        ...resource,
+        type,
+        url: get('url', resource)
```

Comment:
should we throw an error here if there is no link ? ( to be shown in DataDog or else ) 
Just a suggestion

Reply:
Those checks have been made before by cockpit and the mooc.

Reply:
OK ! marked as resolved then !